calico/felix/felix.py

```diff
@@ -411,9 +393,7 @@ def handle_endpointdestroyed(self, message, sock):
             return
 
         # Unsubscribe endpoint.
-        sock = self.sockets[Socket.TYPE_ACL_SUB]
-        sock._zmq.setsockopt(zmq.UNSUBSCRIBE, delete_id.encode('utf-8'))
-
+        self.sockets[Socket.TYPE_ACL_SUB].unsubscribe(delete_id.encode('utf-8'))
```

Comment:
Separately, I believe this function is bugged. If we're asked to destroy an endpoint we don't think exists, we should either reply with an error code or resync. Currently, as far as I can see, we do neither of those things, we just finish executing.